src/mappings/PoolRegistry.ts

```diff
@@ -36,6 +36,28 @@ export function handlePoolUpdated(call: PoolUpdated): void {
     call.params.name,
     call.params.data,
   ])
+  let poolId = call.params.pool.toHexString()
+  let hash = call.params.data
+  let oldPoolAddresses = PoolAddresses.load(poolId)
+
+  if (oldPoolAddresses == null) {
+    log.error('handlePoolUpdated: could not load old pool addresses', [])
+    return
```

Comment:
Should this not created the handlers anyways? There is no cost to us to create them, right?